wasm/__init__.py

```diff
@@ -2106,7 +2107,7 @@ def spec_memorygrow(config):
     if sz + n == len(mem["data"]) // 65536:  # success
         config["operand_stack"].append(sz)
     else:
-        config["operand_stack"].append(2 ** 32 - 1)  # put -1 on top of stack
+        config["operand_stack"].append(constants.UINT32_MAX)  # put -1 on top of stack
```

Comment:
According to the comment, this is really to be interpreted as `INT32_NEGATIVE_ONE`. Using `UINT32_MAX` confused me.